app/models/mixins/ansible_authentications.rb

```diff
@@ -80,4 +69,11 @@ def authentication_for_type(auth_type, name)
     auth.authtype = auth_type
     auth
   end
+
+  def update_ansible_authentication(auth, attributes)
+    attributes.each do |attribute, value|
```

Comment:
Minor nit, we might want to have a different local variable since `attributes` method from activerecord would also be defined here.